src/modules/Profile.rb

```diff
@@ -15,10 +15,13 @@ class ProfileClass < Module
     GENERIC_PROFILE_SECTIONS = [
       # AutoYaST has its own partitioning
       "partitioning",
+      "partitioning_advanced",
       # AutoYaST has its Preboot Execution Environment configuration
       "pxe",
       # Flags for setting the solver while the upgrade process with AutoYaST
-      "upgrade"
+      "upgrade",
+      # Flags for controlling the update backups (see Installation module)
+      "backup"
```

Comment:
Yes, and that's the reason why it's not here. When I was implementing the first design, I used the docs.